examples/linear_bandits_config.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
I am trying to change the config style such that a bandit will accept all environment parameters. In this way, we do not need to write a new program to parse the environment parameters. Feel free to change the `main.py` file. I will take care of the left changes associated with other bandit settings.

```
"environment": {
	"type": "CorrelatedBandit",
	"CorrelatedBandit": {
		"arm": {
			"type": "CorrelatedArm",
			"means": [[0.3,0.2], [0.5,0.6], [0.7,0.9]]
		},
		"param": [0.5,0.5]
	}
}
```

Reply:
I have already updated the `main.py` file such that arm setup in now in bandit initialization.